env/build-dep.yml

```diff
@@ -36,7 +36,7 @@
         - libgrpc++-dev
         - libprotobuf-dev
         - protobuf-compiler-grpc
-      when: ansible_distribution == 'Ubuntu' and ansible_lsb.major_release|int >= 18
+      when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu' and ansible_lsb.major_release|int >= 18
```

Comment:
minor: I had to hunt for a claim that `and` has higher precedence than `or` (I found one here: https://riptutorial.com/ansible/example/12269/when-condition). Maybe use explicit parentheses to avoid the need to check for this... 😺

Reply:
Oh. you're not the first one who asked about the precedence. So it looks like this statement deserve parentheses :)

Reply:
(Just curious, is there any well-known language in which `and` and `or` have the same precedence?)